### ISSUE TITLE
Function not found HIGH_LEVEL_CALL

### ISSUE THREAD

**Issue by kendricktan** [crytic/slither#784]
Hi, thanks for the great tool.

I found out that slither throws a `NoneType` error when library function call composition is performed in a specific sequence of `for`, and logical statements.

```bash
slither --version
0.7.0

python --version
3.8.5

node --version
v14.15.1
```

## MVP Repository (w/ hardhat)
https://github.com/kendricktan/slither-library-bug

**Comment by montyly** [comment]
Hi @kendricktan 

Thank you for reporting this, and for creating the MVP (this is super useful). I will investigate why it happens

**Comment by montyly** [comment]
Hi @kendricktan , the issue has been fixed with the [0.7.1 release](https://github.com/crytic/slither/releases/tag/0.7.1). 

Thanks again for creating the PoC, it helped us to fix the bug
